packages/gatsby-cli/src/util/configstore.ts

```diff
@@ -0,0 +1,56 @@
+import Configstore from "configstore"
+import prompts from "prompts"
+import report from "../reporter"
+
+interface IConfigStore {
+  settings?: {
+    [key: string]: string
+  }
+  get(key: string): string
+  set(key: string, value: string): void
+}
+
+let conf: IConfigStore
+try {
+  conf = new Configstore(`gatsby`, {}, { globalConfigPath: true })
+} catch (e) {
+  // This should never happen (?)
+  conf = {
+    settings: {
+      "cli.packageManager": ``,
+    },
+    get: (key: string): string => conf.settings![key],
+    set: (key: string, value: string): void => {
+      conf.settings![key] = value
+    },
+  }
```

Comment:
It seems that this should never happen.`Configstore` is also used in gatsby-core-utils, but try-catch is not used. I would like to delete these lines.

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-core-utils/src/get-config-store.ts

Reply:
Yeah, looked at configstore's code. It doesn't ever throw on instantiation. Let's use the `gatsby-core-utils` version and drop the try/catch stuff